scriptmodules/supplementary/raspbiantools.sh

```diff
@@ -77,7 +77,7 @@ function install_firmware_raspbiantools() {
     for pkg in "${pkgs[@]}"; do
         if hasPackage "$pkg" "$ver" ne; then
             deb="${pkg}_${ver}_armhf.deb"
-            if ! wget -O"$deb" "$url/$deb"; then
+            if download "$url/$deb"; then
```

Comment:
shouldn't this be `! download "$url/$deb"` ?

Reply:
Yes. Thanks.

Reply:
Fixed thanks.